docs/quickstart.md

```diff
@@ -128,8 +128,10 @@ The following step will partition and format your disk, and mount it to `/mnt`.
 
 **Please note: This will erase any existing data on your disk.**
 
-```console
+```sh
 sudo nix --experimental-features "nix-command flakes" run github:nix-community/disko -- --mode disko /tmp/disk-config.nix
+# Or use nix-shell. Be aware that this might run an older version of disko!
```

Comment:
```suggestion
# Or use the potentially older version from nixpkgs. When reporting github issues, make sure though that you also test with the latest version from this github repository!
```

Reply:
I think if we're just going to tell the users to run the latest version of disko from this repository if issues come up, then I don't think we should add the `nix-shell -p disko` command to the docs